src/lib/wallet/GoodWalletClass.js

```diff
@@ -386,12 +386,12 @@ export class GoodWallet {
         return true
       }
 
-      const runRes = Promise.race([run(), delay(5000, false)])
+      const runRes = Promise.race([run(), delay(10000, false)])
```

Comment:
No, this is an attempt for preventing an error.
This is just how long it tries to fetch data from the api's.
If it takes longer then the ms set, it breaks off the race.
Reason why I increased is because 40-50% of the sentry reports showed the error of 'pollEvents not completed after 5 seconds'